README.md

```diff
@@ -222,6 +237,36 @@ AmioWebchatClient.onMessageEcho((data) => {
 })
 ```
 
+### onNotificationReceived(func)
+Sets a callback function that will be called every time a notification is received from server.
+
+Parameters:
+- **func** - Function. It should accept one parameter which contains the notification payload. The payload can be any valid JSON element (string, object, number...).
+
+Example usage:
+```js
+AmioWebchatClient.onNotificationReceived((payload) => {
```

Comment:
in `amio-sdk-js` we use **the 1st letter small** for services that are **not** to be instantiated. I think we should be consistent here as well. Or do you think it's better this way?